src/controllers/AuthController.js

```diff
@@ -105,6 +105,146 @@ class AuthController {
       picture: user.photos[0].value,
     };
   }
+
+  /**
+   * signup only for organizations
+   *
+   * @static
+   * @param {*} req
+   * @param {*} res
+   * @returns {object} res
+   * @memberof Auth
+   */
+  static async signup(req, res) {
+    const hashedPassword = encrypt.hashPassword(req.body.password);
+    const { companyName, username, email } = req.body;
+
+    const ret = field =>
+      res.status(400).json({ message: `${field} already exist` });
+
+    const existUsername = await User.findOne({
```

Comment:
@Karlmusingo Should be checked from a middleware. Try to extract the finding a user to another file